.NET/Microsoft.Recognizers.Text.Choice/Arabic/Extractors/ArabicBooleanExtractorConfiguration.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+using Microsoft.Recognizers.Definitions.Arabic;
+
+namespace Microsoft.Recognizers.Text.Choice.Arabic
+{
+    public class ArabicBooleanExtractorConfiguration : IBooleanExtractorConfiguration
+    {
+        public static readonly Regex TrueRegex =
+            new Regex(ChoiceDefinitions.TrueRegex, RegexOptions.Singleline);
+
+        public static readonly Regex FalseRegex =
+            new Regex(ChoiceDefinitions.FalseRegex, RegexOptions.Singleline);
```

Comment:
Is it by coincidence that everything works right without telling the regexes to match right-to-left or is the behaviour indeed correct?
Choice is simpler, but in other types, especially DateTime, this will probably lead to issues.

Reply:
I added `RegexOptions.RightToLeft` to regex options.